javascript-data-model/bookshelf.js

```diff
@@ -0,0 +1,79 @@
+var bookshelf = [
+  {
+    isbn: '978-1491929483',
+    title: 'Introduction to JavaScript Object Notation: A To-the-Point Guide to JSON',
+    author: 'Lindsay Bassett'
+  },
+  {
+    isbn: '978-1484218624',
+    title: 'JSON Quick Syntax Reference',
+    author: 'Wallace Jackson'
+  },
+  {
+    isbn: '978-0692232699',
+    title: 'Build APIs You Won\'t Hate: Everyone and their dog wants an API, so you should probably learn how to build them',
+    author: 'Phil Sturgeon & Laura Bohill'
+  }
+];
+
+console.log('The name of the author of the second book is:', bookshelf[1].author);
+console.log('The ISBN of the first book is:', bookshelf[0].isbn);
+console.log('The title of the third book is:', bookshelf[2].title);
+
+var orderHistory = [
+  {
+    type: 'Book',
+    name_of_product: 'The Art of Sql',
+    author: 'Stephane Faroult',
+    date_placed: 'July 3rd 2020',
+    date_delivered: 'July 5th 2020',
+    return_window: 'August 4th 2020',
+    ship_to: 'JS Masher',
+    order_num: 2648248,
+    price: 33.99
+  },
```

Comment:
All the data for the item in the order is intermixed with the data about the order itself, which is confusing. You should organize the data for each item and store it into a property on the object representing the order.